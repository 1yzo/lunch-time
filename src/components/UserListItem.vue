<template>
    <div>
        <div class="user-list-item">{{ name }}</div>
    </div>
</template>

<script>
export default {
    props: {
        name: String
    }
}
</script>

<style lang="scss">
.user-list-item {
    padding: 20px;
    border-bottom: 2px solid black;
}
</style>

