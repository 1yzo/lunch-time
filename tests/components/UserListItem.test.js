import Vue from 'vue';
import { shallowMount } from '@vue/test-utils';
import UserListItem from '../../src/components/UserListItem';
import { coffeeUsers } from '../fixtures/users';

describe('UserListItem', () => {
    it('renders correctly', () => {
        const wrapper = shallowMount(UserListItem, {
            propsData: {
                user: coffeeUsers[0]
            }
        });
        expect(wrapper.element).toMatchSnapshot();
    });
});