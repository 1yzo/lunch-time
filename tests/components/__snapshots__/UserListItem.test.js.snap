// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserListItem renders correctly 1`] = `
<div>
  <div
    class="user-list-item"
  >
    <div>
      first
    </div>
     
    <div
      style="display: flex;"
    >
      <!---->
       
      <!---->
       
      <!---->
    </div>
  </div>
</div>
`;
